lib/client.js

```diff
@@ -391,6 +394,254 @@ Client.prototype.batchRead = function (records, policy, callback) {
+ *       }),
+ *     udf: {
+ *          module: 'udf',
+ *          funcname: 'function2',
+ *          args: [[1, 2, 3]]
+ *          }
+ *    }
+ * ]
+ * Aerospike.connect(function (error, client) {
+ *   if (error) throw error
+ *   client.batchApply(batchRecords, udf, function (error, results) {
+ *     if (error) throw error
+ *     results.forEach(function (result) {
+ *       console.log(result)
+ *     })
+ *   })
+ * })
+ */
+Client.prototype.batchApply = function (records, udf, batchPolicy, batchApplyPolicy, callback) {
+  if (typeof batchPolicy === 'function') {
+    callback = batchPolicy
+    batchPolicy = null
+    batchApplyPolicy = null
+  } else {
+    if (typeof batchApplyPolicy === 'function') {
+      callback = batchApplyPolicy
+      batchApplyPolicy = null
+    }
+  }
+
+  const cmd = new Commands.BatchApply(this, [records, udf, batchPolicy, batchApplyPolicy], callback)
+  return cmd.execute()
+}
+
+/**
+ * @function Client#batchRemove
+ *
+ * @summary Remove multiple records.
+ *
+ * @description
+ *
+ * This method allows multi sub-commands for each key in the batch.
+ * This method requires server >= 6.0.0.
+ *
+ * @param {object[]} records - List of batch sub-commands to perform.
+ * @param {BatchPolicy} [batchPolicy] - The Batch Policy to use for this operation.
+ * @param {BatchRemovePolicy} [batchRemovePolicy] Remove policy configuration parameters.
+ * @param {batchRecordsCallback} [callback] - The function to call when
+ * the operation completes, with the results of the batch operation.
+ *
+ * @returns {?Promise} - If no callback function is passed, the function
+ * returns a Promise that resolves to the results of the batch operation.
+ *
+ * @since v4.1.0
+ *
+ * @example
+ * const Aerospike = require('aerospike')
+ * const batchType = as.batchTypes
+ * var batchRecords = [
+ *   { type: batchType.BATCH_REMOVE,
+ *     key: new Aerospike.Key('test', 'demo', 'key5'),
+ *     policy: new Aerospike.BatchRemovePolicy({
+ *         filterExpression: exp.eq(exp.binInt('i'), exp.int(37)),
+ *         key: Aerospike.policy.key.SEND,
+ *         commitLevel: 2,
+ *         gen: Aerospike.policy.gen.EQ,
+ *         durableDelete: true
+ *       })},
+ *    },
+ *   { type: batchType.BATCH_REMOVE,
+ *     key: new Aerospike.Key('test', 'demo', 'key6'),
+ *     policy: new Aerospike.BatchRemovePolicy({
+ *         filterExpression: exp.eq(exp.binInt('i'), exp.int(37)),
+ *         key: Aerospike.policy.key.SEND,
+ *         commitLevel: 2,
+ *         gen: Aerospike.policy.gen.EQ,
+ *         durableDelete: true
+ *       })},
+ *    }
+ * ]
+ * Aerospike.connect(function (error, client) {
+ *   if (error) throw error
+ *   client.batchRemove(batchRecords, function (error, results) {
+ *     if (error) throw error
+ *     results.forEach(function (result) {
+ *       console.log(result)
+ *     })
+ *   })
+ * })
+ */
+Client.prototype.batchRemove = function (records, batchPolicy, batchRemovePolicy, callback) {
```

Comment:
Likewise, what can I do using the `batchRemove` that I can't also do using `batchWrite`? Why would I use one vs the other? I think the docs should answer that question.